widget/entry.go

```diff
@@ -21,6 +21,9 @@ import (
 const (
 	bindIgnoreDelay = time.Millisecond * 100 // ignore incoming DataItem fire after we have called Set
 	multiLineRows   = 3
+
+	desktopDoubleClickDelay = 300 // ms - keep in sync with internal/driver/glfw/window.go
```

Comment:
Can't we access the driver constants from here to save the need to keep them in sync manually?

Reply:
As it's buried in the internal package it doesn't really make much difference :) I'd probably just export them where they are for now, I don't think there is a need for a new package - and we don't really add them unless needed or much cleaner.